db/seeds.rb

```diff
@@ -1,8 +1,13 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 
-products = Product.create([{ name: "Spice Cake", unit_price_in_cents: 2200,}, { name: "Pullman Loaf", unit_price_in_cents: 800 }, { name: "Muffin", unit_price_in_cents: 300 }, { name: "Avocado Cake", unit_price_in_cents: 3800 }, {
```

Comment:
there are only 5 prices that were given to us so I can do that real quick. Every other order on the 21 pages of orders will have to be manually put in by somebody too